source/wordle_word_suggester.cpp

```diff
@@ -389,51 +430,57 @@ words_t trim_words_by_user_inputs(std::map<std::string, colored_buckets_t>& answ
         std::cout << "--Enter your guess and color result on two separate lines \n";
         std::getline (std::cin, guess_str);
         std::getline (std::cin, result_str);
+
+        words_t trimmed_words;
         if (first_guess)
-            words = answers[guess_str][result_str];
+        {
+            if (guess_str.empty())
+                guess_str = "slate"; // Assume the word input is "slate" if the first line is blank
```

Comment:
Slate isn't the first word I use anymore, but the one I've been using for months won't stay forever either. Maybe remove this feature? At least wait until I update the algorithm to find a new first word

Reply:
I made the change from slate to parse